dockerfiles/platform-gnn-gpu/Dockerfile

```diff
@@ -20,43 +20,51 @@
 # Others may not be wanted at all in container mode - tensoflow and nvidia  example??
 
 ARG BUILD_LABEL
-ARG DOCKER_REPO
-FROM ${DOCKER_REPO}/platform-core:${BUILD_LABEL}
+FROM hpccsystems/platform-core:${BUILD_LABEL}
```

Comment:
What's the reasoning behind removing DOCKER_REPO and allowing us to target a custom platfom-core build?

Reply:
fixed